Translating to Russian language. (third addition)

SolidSnacke opened the pull request:
I finished the translation into Russian a little more as best I could.

SolidSnacke committed:
Translating to Russian language. (third addition)

I finished the translation into Russian a little more as best I could.

Cohee1207 commented on public/index.html:
```diff
@@ -3044,7 +3045,7 @@ <h3 class="margin0">
                             </div>
                             <small data-i18n="or">or</small>
-                            <select id="world_editor_select" class="margin0">
+                            <select id="world_editor_select" class="widthNatural flex1 margin0">
```
In English these classes cause the select to overexpand

SolidSnacke replied:
I know, I just, albeit crookedly, solved the problem for my language. I changed this place because the words in Russian in those places did not fully fit. I just didn't know how critical the change would be for you in these places. If there is another, nicer way to fix this, I'd be interested in seeing it.

Cohee1207 replied:
You can either remove `---` or shorten the word (e.g. `Изменить` vs `Редактировать`)

SolidSnacke replied:
Okay, what about that list box that comes after the search string? Some words don't fit there either. Is it possible to edit it somehow so that it has a fixed length? Because I also applied the widthNatural flex1 margin0 parameter to it.

Cohee1207 replied:
It's okay if some UI elements will wrap, the layout is not fixed to just one row.

SolidSnacke replied:
Okay, I get it. In the next commit I will roll back the changes to these lines.

SolidSnacke committed:
Addendum number four.

I reached for everything that most often caught my eye.
I also replaced widthNatural flex1 margin0 with widthNatural margin0. Now there should be no problem with fields taking up extra space. Although perhaps you asked me to return everything as it was.

Cohee1207 commented on public/index.html:
```diff
@@ -4576,9 +4584,9 @@ <h4>
-                                <label for="group">
-                                    <small data-i18n="">Inclusion Group</small>
-                                </label>
+                                <small for="group" data-i18n="Inclusion Group">
```
`for` attribute is not valid for `small` element.

SolidSnacke replied:
It seems like I returned it the way it was. I'll comment now.

Cohee1207 commented on public/index.html:
```diff
@@ -4435,7 +4443,7 @@ <h4>
                                         <label for="position" class="WIEntryHeaderTitleMobile" data-i18n="Position:">Position:</label>
                                         <select name="position" class="widthNatural margin0" data-i18n="[title]T_Position" title="↑Char: Before Character Definitions&#13;↓Char: After Character Definitions&#13;↑AN: Before Author's Note&#13;↓AN: After Author's Note&#13;@D: at Depth&#13;">
-                                            <option value="0" data-i18n="[title]T_Position" title="↑Char: Before Character Definitions&#13;↓Char: After Character Definitions&#13;↑AN: Before Author's Note&#13;↓AN: After Author's Note&#13;@D: at Depth&#13;"><span Before Char Defs>↑Char</label></option>
+                                            <option value="0" data-i18n="[title]T_Position" title="↑Char: Before Character Definitions&#13;↓Char: After Character Definitions&#13;↑AN: Before Author's Note&#13;↓AN: After Author's Note&#13;@D: at Depth&#13;"><span data-i18n="Before Char Defs">↑Char</label></option>
```
There's a `</label>` closing tag instead of span. I think that's a mistake (possibly even before this PR).

SolidSnacke replied:
I tried to translate pop-up windows and posts, but as I understand it is impossible, I need to translate one thing. Either I don't have enough knowledge.

SolidSnacke replied:
I forgot to correct it, my fault.

Cohee1207 commented on public/index.html:
```diff
@@ -3058,7 +3066,7 @@ <h3 class="margin0">
                             <div id="world_popup_delete" class="menu_button fa-solid fa-trash-can redWarningBG" title="Delete World Info" data-i18n="[title]Delete World Info"></div>
                             <input type="search" class="text_pole textarea_compact" data-i18n="[placeholder]Search..." id="world_info_search" placeholder="Search...">
-                            <select id="world_info_sort_order" class="widthNatural flex1 margin0">
+                            <select id="world_info_sort_order" class="widthNatural margin0">
```
I still think it's better not to use widthNatural here.

SolidSnacke replied:
May I know why? I just don’t quite understand what the problem is, since I’m not a big expert in the html language.

Cohee1207 replied:
Nevermind, looked at the wrong commit.

SolidSnacke replied:
I see, then I have no questions about it.

SolidSnacke committed:
Small fix.

SolidSnacke commented:
I apologize for wasting your time checking all this.
